packages/api/src/lib/suggestedOperations.ts

```diff
@@ -0,0 +1,118 @@
+ *
+ * The criteria for a suggested endpoint are the following:
+ *  - GET request
+ *  - Has no required query or body parameters
+ *  - Endpoint has not been deprecated
+ *
+ * @param oas
+ */
+export function getSuggestedOperation(oas: Oas) {
+  const suggested: Operation[] = [];
+
+  Object.values(oas.getPaths()).forEach(path => {
+    Object.values(path).forEach(operation => {
+      // We don't really support webhooks with this library so if this operation is one we shouldn't
+      // return it as a recommended operation.
+      if (operation instanceof Webhook) {
+        return;
+      }
+
+      if (operation.isDeprecated()) {
+        return;
+      }
+
+      if (operation.method.toLowerCase() === 'get') {
+        const hasRequiredParameters = operation.hasRequiredParameters();
+        const hasRequiredRequestBody = operation.hasRequiredRequestBody();
+
+        // If the criteria matches, push it into a recommended array
+        if (!hasRequiredParameters && !hasRequiredRequestBody) {
+          suggested.push(operation);
+        }
+      }
+    });
+  });
+
+  if (suggested.length) {
+    return suggested[0];
+  }
+
+  return false;
+}
+
+/**
+ * Generate an example code snippet for a given (suggested) operation. We'll show this to users
+ * post-codegeneration so they can see how to use the SDK we created for them.
+ *
+ */
+export async function buildCodeSnippetForOperation(oas: Oas, operation: Operation, opts: { identifier: string }) {
+  // If this endpoint has authentication on it then we should try to flesh out some placeholder
+  // values in the `.auth()` SDK method for them so they can see how to use auth.
+  let auth: AuthForHAR = {};
+  const hasAuth = !!operation.getSecurity()?.length;
+  if (hasAuth) {
+    operation.getSecurityWithTypes().forEach(schemes => {
+      if (!schemes) return;
+      schemes.filter(Boolean).forEach(scheme => {
+        if (!scheme) return;
+
+        // eslint-disable-next-line no-underscore-dangle
+        const schemeName = scheme.security._key;
+        if (scheme?.type === 'Basic') {
+          auth[schemeName] = { user: 'username', pass: 'password' };
+        } else {
+          auth[schemeName] = 'token';
+        }
+      });
+    });
+
+    auth = oas.getAuth(auth) as AuthForHAR;
+  }
+
+  // We're pulling in `@reamde/oas-to-har` and `httpsnippet-client-api` here instead of using
+  // `@readme/oas-to-snippet`, which would handle both, because we don't need the entire
+  // `oas-to-snippet` for what we're doing here. All we want to do is generate a very simple code
+  // example for `api` snippets and because we're controlling which kinds of endpoints we're
+  // generating this for the HAR dataset we're working with here is mostly a fully known object.
+  const har = oasToHar(oas, operation, undefined, auth)?.log?.entries?.[0]?.request;
+  if (!har) {
+    return false;
+  }
+
+  const snippet = client.convert(
```

Comment:
I don't love this but I also **really** don't want to have to make `@readme/oas-to-snippet` a dependency because that depends on `httpsnippet-client-api` and if we need to make fixes to this snippet generation work it's be a multistep process:

1. publishing `httpsnippet-client-api`
2. pulling that release into `@readme/oas-to-snippet`
3. publishing `@readme/oas-to-snippet`.
4. puling that release into `api`
5. publishing `api`.